pjlib/include/pj/config_site_sample.h

```diff
@@ -389,7 +389,10 @@
 
     /* Fine tune Speex's default settings for best performance/quality */
     #define PJMEDIA_CODEC_SPEEX_DEFAULT_QUALITY 5
-    
+
+    /* Software clock for sound device can improve quality */
```

Comment:
Add more explanation about what quality improvement that's expected, such as: Using software clock for (audio) media flow can improve quality (more consistent RTP timing and less jitter/burst).